README.md

```diff
@@ -23,6 +23,8 @@ When you want to add an event to [hacken.in](https://hacken.in), we feature the
 * Electronics and Robots
 * Networking
 
+If you want to help us in extending the calendar to your city, open an issue, so we can get you started!
```

Comment:
→ `…to your city, [open an issue](https://github.com/hacken-in/hacken-in/issues/new), so we can…`